gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/management-openapi-v2.yaml

```diff
@@ -1617,6 +1617,27 @@ paths:
                 default:
                     $ref: "#/components/responses/Error"
 
+    # API Documentation
+    /environments/{envId}/apis/{apiId}/pages:
```

Comment:
**question**: Should it be paginated?

Reply:
Can be added later. WDYT ?

Reply:
No pb, it was just a question to be sure we don't forget it :)